rest/model/stats.go

```diff
@@ -39,7 +40,58 @@ func (apiStatus *APIStats) BuildFromService(h interface{}) error {
 	return nil
 }
 
-// ToService returns a service layer distro using the data from APIStats.
-func (apiStatus *APIStats) ToService() (interface{}, error) {
-	return nil, errors.Errorf("ToService() is not implemented for APIStats")
+// ToService returns a service layer distro using the data from APITaskStats.
+func (apiStatus *APITaskStats) ToService() (interface{}, error) {
+	return nil, errors.Errorf("ToService() is not implemented for APITaskStats")
+}
+
+// APIHostStatsByDistro is a slice of host stats for a distro
+// the 3 structs below are nested within it
+type APIHostStatsByDistro struct {
+	Distros []APIHostStatsForDistro `json:"distros"`
+}
+
+type APIHostStatsForDistro struct {
+	Distro string                   `json:"distro"`
+	Stats  APISingleDistroHostStats `json:"stats"`
+}
+
+type APISingleDistroHostStats struct {
+	Status string                   `json:"status"`
+	Counts APISingleStatusHostStats `json:"counts"`
+}
```

Comment:
I think in some of the other routes we have fewer things public for nested documents?

Reply:
made most of them private